src/components/Checkbox/Checkbox.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../themes/nice/nice.scss';
+
+.Checkbox {
+    margin: auto;
+    user-select: none;
+    cursor: pointer;
+    display: block;
+
+    &__text {
```

Comment:
Why not `&__label`?

Reply:
so not to confuse with the `<label>` element. This is a span. but I can change